src/core/client/admin/components/PaginatedSelect/PaginatedSelect.tsx

```diff
@@ -66,42 +96,58 @@ const PaginatedSelect: FunctionComponent<Props> = ({
               )}
             </Dropdown>
           </IntersectionProvider>
-        </ClickOutside>
-      )}
-    >
-      {({ toggleVisibility, ref, visible }) => (
-        <Button
-          className={cn(styles.button, className)}
-          variant="flat"
-          adornmentLeft
-          color="mono"
-          onClick={toggleVisibility}
-          ref={ref}
-          uppercase={false}
-        >
-          {Icon && (
-            <ButtonSvgIcon className={styles.buttonIconLeft} Icon={Icon} />
-          )}
-          <Flex alignItems="center" className={styles.wrapper}>
-            {selected}
+        )}
+      >
+        {({ ref }) => (
+          <Flex
+            className={cn(styles.button, className)}
+            onClick={() => setIsPopoverVisible(true)}
+            ref={ref}
+            justifyContent="space-between"
+            aria-label={label}
+          >
+            {Icon && (
+              <ButtonSvgIcon className={styles.buttonIconLeft} Icon={Icon} />
+            )}
+            {isPopoverVisible && !!onFilter ? (
+              <Localized
+                id="admin-paginatedSelect-filter"
+                attrs={{ "aria-roledescription": true, "aria-label": true }}
```

Comment:
Looks like this `Localized` is only for the `TextArea` which only has an `aria-label` attribute and not an `aria-roledescription` (that's the button below)? Can you take a look?

Reply:
✅